python/nav/mibs/entity_mib.py

```diff
@@ -277,8 +277,6 @@ def get_chassis_of(self, entity):
     def clean(self):
         """Cleans the table data"""
 
-        if sys.version_info[0] == 2:  # Python 2 only
-            self._clean_unicode()
```

Comment:
Unless I'm mistaken, you can take out the entire `_clean_unicode()` method as well, as it doesn't seem to be called from anything else.